engine-rest/engine-rest/pom.xml

```diff
@@ -92,6 +92,18 @@
       <version>1.6.5</version>
       <scope>test</scope>
     </dependency>
+    
+    <!--
+      needed for offline shrinkwrap (in particular CustomJacksonDateFormatTest), 
+      BOM must be installed before this module is built
+    -->
+    <dependency>
+      <groupId>org.camunda.bpm</groupId>
+      <artifactId>camunda-bom</artifactId>
+      <version>${project.version}</version>
+      <type>pom</type>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Just to understand this correctly:
This dependency is included so this project is definitely build after the BOM project?
Shouldn't this already be the case because
1. this project's parent is `rest-root`
2. `rest-root`'s parent is `database-settings`
3. `database-settings`' parent is `parent`
4. `parent` depends on BOM to be build

Reply:
I though the same, but apparently it is not. You can try this out by removing the dependency here, removing the 7.13.0-SNAPSHOT bom from your local Maven repo and then building the reactor in offline mode (`-o`, so that the bom won't be fetched from remote). `org.camunda.bpm.engine.rest.CustomJacksonDateFormatTest` should fail then.

My theory is the following:

* The bom pom is in the reactor so it can be resolved for imports and exports already before the bom module is built.
* The test uses Shrinkwrap. Shrinkwrap seems to resolve the bom from the local Maven repo.
* As the bom is not properly built yet, it is also not installed into the Maven repo.

That said, I have no documentation at hand supporting this claim :)